feat(rdb): remove endpoints (#2944)
scaleway/docs-content: managed-databases/postgresql-and-mysql/how-to/remove-public-endpoint.mdx

tags: managed-database postgresql mysql endpoints database-instance
dates:
  validation: 2024-03-22
  posted: 2024-03-22
categories:
  - managed-databases
  - postgresql-and-mysql
---

When you create a Database Instance, a public endpoint is automatically attributed to it.

For Read Replicas, you can choose whether they take on a public or private endpoint.

If you are using a public endpoint with your Database Instances or Read Replicas, you can remove them anytime in the Scaleway console.

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)

## From Database Instances

1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Scroll to the **Network** section.
4. Click **Remove** next to **Public endpoint**. A pop-up appears.
    <Message type="important">

>       When you remove a public endpoint your Database Instance becomes inaccessible over the internet.

Reviewer: ```suggestion
      When you remove a public endpoint, your Database Instance becomes inaccessible over the internet.
```

    </Message>
5. Click **Remove public endpoint** to do so.

    Your Database Instance re-initializes.

<Message type="tip">
  Click **Add public endpoint** in the **Network** section to re-add a public endpoint. When you add a new public endpoint, a new IPv4 address and port will be generated. Make sure you update the configuration of your applications with the new endpoint.
</Message>

## From Read Replicas

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- A [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)

1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Scroll down to the **Read Replica** section.
4. Click <Icon name="more" /> > **Remove public endpoint**. A pop-up appears.
    <Message type="important">

>       When you remove a public endpoint your Read Replica becomes inaccessible over the internet.

Reviewer: ```suggestion
      When you remove a public endpoint, your Read Replica becomes inaccessible over the internet.
```

    </Message>
5. Click **Remove public endpoint** to do so.

    Your Read Replica re-initializes.

<Message type="tip">

>   Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will stay the as the previous.

Reviewer: ```suggestion
  Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will remain the same as the previous one.
```

---
meta:
  title: How to remove a public endpoint
  description: This page explains how to remove the public endpoint of your Database Instance or Read Replica
content:
  h1: How to remove a public endpoint
  paragraph: This page explains how to remove the public endpoint of your Database Instance or Read Replica
tags: managed-database postgresql mysql endpoints database-instance
dates:
  validation: 2024-03-22
  posted: 2024-03-22
categories:
  - managed-databases
  - postgresql-and-mysql
---

When you create a Database Instance, a public endpoint is automatically attributed to it.

> For Read Replicas you can choose whether they take on a public or private endpoint.

Reviewer 2: ```suggestion
For Read Replicas, you can choose whether they take on a public or private endpoint.
```

If you are using a public endpoint with your Database Instances or Read Replicas, you can remove them anytime in the Scaleway console.

<Macro id="requirements" />

> <Message type="requirement">

Reviewer 2: ```suggestion
```

>   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

Reviewer 2: ```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

>   - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)

Reviewer 2: ```suggestion
- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
```

> </Message>

Reviewer 2: ```suggestion
```

## From Database Instances

1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Scroll to the **Network** section.
4. Click **Remove** next to **Public endpoint**. A pop-up appears.
    <Message type="important">
      When you remove a public endpoint your Database Instance becomes inaccessible over the internet.
    </Message>
5. Click **Remove public endpoint** to do so.

    Your Database Instance re-initializes.

<Message type="tip">
  Click **Add public endpoint** in the **Network** section to re-add a public endpoint. When you add a new public endpoint, a new IPv4 address and port will be generated. Make sure you update the configuration of your applications with the new endpoint.
</Message>

## From Read Replicas

<Macro id="requirements" />

> <Message type="requirement">

Reviewer 2: ```suggestion
```

  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)

>   - You have a [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)

Reviewer 2: ```suggestion
- A [Read Replica](/managed-databases/postgresql-and-mysql/how-to/create-read-replica)
```

> </Message>

Reviewer 2: ```suggestion
```

1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the database name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Scroll down to the **Read Replica** section.
4. Click <Icon name="more" /> > **Remove public endpoint**. A pop-up appears.
    <Message type="important">
      When you remove a public endpoint your Read Replica becomes inaccessible over the internet.
    </Message>
5. Click **Remove public endpoint** to do so.

    Your Read Replica re-initializes.

<Message type="tip">
  Click **Add public endpoint** in the **Read Replica** section to re-add a public endpoint. The IP address of the endpoint will stay the as the previous.
</Message>

> 

Reviewer 2: ```suggestion

```

<Navigation title="See also">
  <PreviousButton to="/managed-databases/postgresql-and-mysql/how-to/connect-database-private-network">How to connect a Database Instance to a Private Network</PreviousButton>
  <NextButton to="/managed-databases/postgresql-and-mysql/how-to/create-read-replica">How to create Read Replicas</NextButton>
</Navigation>

> 

Reviewer 2: ```suggestion
```
